mfr/extensions/tabular/templates/viewer.mako

```diff
@@ -78,7 +78,8 @@
                 var filtered = false;
                 var item = data[i];
                 for (var title in item) {
-                    if (search !== "" && item[title].toString().indexOf(search) !== -1) {
+                    var re = new RegExp(search, 'i');
+                    if (search !== "" && item[title].toString().match(re)) {
```

Comment:
Use single quotes for string constants (including empty string).
https://github.com/felixge/node-style-guide#use-single-quotes

(citation by way of [cosdev](http://cosdev.readthedocs.org/en/latest/style_guides/javascript.html))